packages/widgetbook/lib/src/knobs/knobs_builder.dart

```diff
@@ -10,6 +11,15 @@ abstract class KnobsBuilder {
     bool initialValue = false,
   });
 
+/// Creates a textfield that can be typed in and optionally hold a
+// color value
+
```

Comment:
This should not have an empty space between the comment and the code. This should be a docs comment which starts with `///` instead of `//` and the indentation should match the indentation of the code.

Reply:
Got it